### ISSUE TITLE
[iOS] checkInternetConnection returns false because of missing network permissions (info.plist)

### ISSUE THREAD

**Issue by MaxInMoon** [rgommezz/react-native-offline#125]
## Problem

`checkInternetConnection` always returns `false` on iOS (both simulator and device) using `checkInternetConnection`

```js
import { checkInternetConnection } from 'react-native-offline';
const isConnected = await checkInternetConnection();
```

## Resolution

The "App Transport Security Settings" section in the `info.plist` file must authorize the ping to google. Two options

**Development**

> Allow Arbitrary Loads-> YES

<img width="523" alt="capture d ecran 2018-09-20 a 17 26 00" src="https://user-images.githubusercontent.com/26519101/45829094-6a88db80-bcfa-11e8-812e-6805eadb5c77.png">

**Production**

List `google.com` into `Exception Domains` with `NSTemporaryExceptionAllowsInsecureHTTPLoads ` and `NSTemporaryExceptionAllowsInsecureHTTPLoads ` equal to `YES`

<img width="747" alt="capture d ecran 2018-09-20 a 17 17 31" src="https://user-images.githubusercontent.com/26519101/45829084-6492fa80-bcfa-11e8-97fd-3c95bcd552ee.png">

## Reproduce

Try to remove network permissions listed above, and test the `checkInternetConnection()` call. Don't forget to re-build the app.

## IMPORTANT: update the doc?

I think that it's urgent to add this important detail in the readme.
I lost 2 days by upgrading react-native/redux-saga/babel versions, and messing up with other issues giving the same issue effect (and I think I'm not the only one).

**Comment by MaxInMoon** [comment]
Another solution would be to switch from http to https (?)

**Comment by rgommezz** [comment]
I just released version v3.14.0 that uses `https` protocol across all components
